Formula/precomp.rb

```diff
@@ -0,0 +1,28 @@
+class Precomp < Formula
+  desc "Command-line precompressor to achieve better compression"
+  homepage "http://schnaader.info/precomp.php"
+  url "https://github.com/schnaader/precomp-cpp/archive/v0.4.5.tar.gz"
+  sha256 "39add141554f0186200911ab61838b69f5777956b3dfd37d82bb6923607258dc"
+  head "https://github.com/schnaader/precomp-cpp.git"
+
+  def install
+    # HEAD already has that, but current stable version does not compile with clang without that patch
```

Comment:
So, the diff available through Github is not acceptable (it patches too much other files and those depend on older commits, so the patching fails for many hunks). Is it ok to provide a patch file from my own repository that was created just for that? https://github.com/maki-chan/precomp-cpp-patches